pkg/anonymization/anonymizer.go

```diff
@@ -49,6 +53,12 @@ const (
 		"some data won't be anonymized(ipv4 and cluster base domain). The error is %v"
 )
 
+var (
+	TranslationTableSecretName = "translation"
```

Comment:
I'd name it more precisely, something like obfuscation-translation-table or whatever